Gemfile

```diff
@@ -1,11 +1,14 @@
+# frozen_string_literal: true
 source 'https://rubygems.org'
 
 gemspec
 
 gem 'liquid', github: 'Shopify/liquid', ref: 'master'
 
-
 group :test do
+  gem 'rubocop', require: false
+  gem 'rubocop-performance', require: false
+  gem 'rubocop-shopify', require: false
```

Comment:
Shouldn't we specify a version?  Otherwise, I would expect that updates will cause CI to start failing for this library.

Reply:
Updated. I also noticed that Shopify/ruby-style-guide does not yet support Rubocop 1.0.